ts/routes/image-occlusion/shapes/to-cloze.ts

```diff
@@ -21,6 +21,9 @@ export function exportShapesToClozeDeletions(occludeInactive: boolean): {
     let clozes = "";
     let index = 0;
     let nextOrdinal = 1;
+
+    console.log(shapes);
```

Comment:
Adding this console log seems to fix the issue, so is the issue race condition?

Reply:
I'm afraid for me, the same issue still happens when I update to the latest commit which has this console.log call. I tried 3 times, and it happened on the 1st and 2nd time, but not the 3rd. If you try a few more times, does it ever happen? Could it be something other than a race condition, such as unpredictable dictionary ordering?

Reply:
I didn't understand the underlying issues, so I have changed the implementation, it works as expected. The general idea is takeout ordinal which does not assign to any shape from a list of ordinal and assign to shape.